app/models/bishop.rb

```diff
@@ -0,0 +1,9 @@
+class bishop < ChessPiece

```

Comment:
All classes should start with an uppercase letter

Reply:
Thanks for the help. I updated all the class names